app/views/admin_views.py

```
@@ -613,3 +1033,525 @@ def post(self):
+            data.append({
+                "id": str(t.id),
+                "date": t.date.strftime("%Y-%m-%d"),
+                "total_hours": t.total_hours,
+                "status": t.status,
+                "remarks": t.remarks,
+            })
+
+        return jsonify(data), 200
+
+# Get single timesheet details
+class AdminViewSingleTimesheet(MethodView):
```

Ruff (0.12.2)

Redefinition of unused `AdminViewSingleTimesheet` from line 880

(F811)